cartoframes/data/observatory/catalog/entity.py

```diff
@@ -18,41 +18,81 @@
 
 
 class CatalogEntity(ABC):
+    """
+    This is an internal class the rest of the classes related to the catalog discovery extend.
+
+    It contains:
+      - Properties: `id`, `slug` (a shorter ID)
```

Comment:
Don't know if after a parenthesis we should add a dot `.`, but we're doing it in other parts of the docs